protonvpn_cli/utils.py

```diff
@@ -226,44 +227,8 @@ def wait_for_network(wait_time):
 
 
 def cidr_to_netmask(cidr):
-    netmask = "0"
-    cidr_netmask = {
-        1: "128.0.0.0",
-        2: "192.0.0.0",
-        3: "224.0.0.0",
-        4: "240.0.0.0",
-        5: "248.0.0.0",
-        6: "252.0.0.0",
-        7: "254.0.0.0",
-        8: "255.0.0.0",
-        9: "255.128.0.0",
-        10: "255.192.0.0",
-        11: "255.224.0.0",
-        12: "255.240.0.0",
-        13: "255.248.0.0",
-        14: "255.252.0.0",
-        15: "255.254.0.0",
-        16: "255.255.0.0",
-        17: "255.255.128.0",
-        18: "255.255.192.0",
-        19: "255.255.224.0",
-        20: "255.255.240.0",
-        21: "255.255.248.0",
-        22: "255.255.252.0",
-        23: "255.255.254.0",
-        24: "255.255.255.0",
-        25: "255.255.255.128",
-        26: "255.255.255.192",
-        27: "255.255.255.224",
-        28: "255.255.255.240",
-        29: "255.255.255.248",
-        30: "255.255.255.252",
-        31: "255.255.255.254",
-        32: "255.255.255.255",
-    }
-
-    netmask = cidr_netmask[cidr]
-    return netmask
+    subnet = ipaddress.IPv4Network("0.0.0.0/%d" % cidr)
```

Comment:
Would you mind changing this to the `.format()` string method instead of %-formatting, so it's consistent with the rest of the code?